sjb/generated/ami_build_origin_int_fedora_crio.xml

```diff
@@ -102,12 +104,16 @@ oct configure aws-client &#39;keypair_name&#39; &#39;libra&#39;
 oct configure aws-client &#39;private_key_path&#39; &#39;/var/lib/jenkins/.ssh/devenv.pem&#39;</command>
         </hudson.tasks.Shell>
         <hudson.tasks.Shell>
-          <command>#!/bin/bash
+          <command>
```

Comment:
Would you mind iterating a tiny bit on the output so we can minimize this diff so it's easier to review?

Reply:
Sure, on it.

Are we fine with `${WORKSPACE}` updates in xmls?

Reply:
WORKSPACE, yes